docs/Queries.md

```diff
@@ -187,23 +187,18 @@ const element = getByA11yValue({ min: 40 });
 
 The interface is the same as for other queries, but we won't provide full names so that they're harder to find by search engines.
 
-### `UNSAFE_ByType`, `ByType`
+### `UNSAFE_ByType`
 
 > Note: added in v1.4
 
-> This method has been **deprecated** and has been prepended with `UNSAFE_` prefix. In react-native-testing-library 2.x only the prefixed version will work.
+> This method has been **deprecated** and is not recommended for use.
```

Comment:
How about:
```suggestion
> This method has been marked unsafe, since it requires knowledge about implementation details of the component. Use responsibly.
```